app/org/sagebionetworks/bridge/services/ConsentService.java

```diff
@@ -160,7 +160,7 @@ public void consentToResearch(Study study, SubpopulationGuid subpopGuid, StudyPa
         activityEventService.publishEnrollmentEvent(participant.getHealthCode(), withConsentCreatedOnSignature);
         
         // Send email, if required.
-        if (sendEmail && participant.getEmail() != null) {
+        if (sendEmail && !subpop.isAutoSendConsentSuppressed() && participant.getEmail() != null) {
```

Comment:
Setting this flag means we send for neither to the participant nor to the study consent notification email. Is this the desired behavior? Should that flag be controlling both?

Reply:
No, it would be better for these to be separate. What I will implement instead is that the provider will assemble the recipients (user, study admin, or both, or neither) and then the SendMailService only sends email to the recipients that are listed by the provider.